modules/core/src/main/java/org/apache/ignite/internal/processors/bulkload/BulkLoadProcessor.java

```diff
@@ -91,15 +103,17 @@ public BulkLoadCacheWriter outputStreamer() {
      * @throws IgniteIllegalStateException when called after {@link #close()}.
      */
     public void processBatch(byte[] batchData, boolean isLastBatch) throws IgniteCheckedException {
-        if (isClosed)
-            throw new IgniteIllegalStateException("Attempt to process a batch on a closed BulkLoadProcessor");
+        try (MTC.TraceSurroundings ignored = MTC.supportContinual(qrySpan)) {
```

Comment:
What's the point of re-supporting same span? Please see my comments for TraceableItrator.